docs/accessibility.md

```diff
@@ -62,9 +62,11 @@ To use, set the `accessibilityHint` property to a custom string on your View, Te
 </TouchableOpacity>
 ```
 
-iOS In the above example, VoiceOver will read the hint after the label, if the user has hints enabled in the device's VoiceOver settings. Read more about guidelines for accessibilityHint in the [iOS Developer Docs](https://developer.apple.com/documentation/objectivec/nsobject/1615093-accessibilityhint)
+#### <div class="label ios">iOS</div>
+In the above example, VoiceOver will read the hint after the label, if the user has hints enabled in the device's VoiceOver settings. Read more about guidelines for accessibilityHint in the [iOS Developer Docs](https://developer.apple.com/documentation/objectivec/nsobject/1615093-accessibilityhint)
```

Comment:
```suggestion
<div class="label ios basic">iOS</div>
In the above example, VoiceOver will read the hint after the label, if the user has hints enabled in the device's VoiceOver settings. Read more about guidelines for `accessibilityHint` in the [iOS Developer Docs](https://developer.apple.com/documentation/objectivec/nsobject/1615093-accessibilityhint)
```

We have an additional class `basic` which removes the skeuomorphic start of label, I think it will work better here. Also wrapping with H4 do no add anything, only produces the weird hash links, let's just skip those tags.

Reply:
I didn't know that existed! Thank you!